### DISCUSSION TITLE
where should I add the users that they can login?

### DISCUSSION THREAD

**Discussion by Root-FTW** [AstraSurge/gpteams#42] [Q&A] [answered]
Thanks to the creator of this project, I was able to launch it in railway, but I must be blind, because I don't see how to add users.

For example, I added this user in the whitelist:

![2023_04_03_17_41_03](https://user-images.githubusercontent.com/108767514/229657196-5a1cfad1-bb26-4b23-9b83-3ba555f87d21.png)

but when he tries to log in he gets this error:
![2023-04-03 17_42_02](https://user-images.githubusercontent.com/108767514/229657246-8905025c-cb17-47db-942f-678954e83af2.png)

am I doing it wrong? where should I add the users that they can login?

**Comment by suikodev** [comment, marked as answer]
Have you checked the system settings -> whitelist again? It's possible that you didn't configure the whitelist correctly.
Example of proper configuration：
![image](https://user-images.githubusercontent.com/37903575/229675919-cfb8ddce-fa5a-45a5-b625-6469a60b2e23.png)
You should see the email address that you added to the whitelist.

**Reply by Root-FTW** [reply]
> Have you checked the system settings -> whitelist again? It's possible that you didn't configure the whitelist correctly. Example of proper configuration： ![image](https://user-images.githubusercontent.com/37903575/229675919-cfb8ddce-fa5a-45a5-b625-6469a60b2e23.png) You should see the email address that you added to the whitelist.

ohh I saw my mistake, I didn't know that after writing the email I had to press the "Enter" key, if you don't press it and just write it and hit update it doesn't work, the correct steps would be

1.- Write the email
2.- Press the enter key
3.- Click on update

THANKS!

**Reply by suikodev** [reply]
I will optimize the operation later.

> > Have you checked the system settings -> whitelist again? It's possible that you didn't configure the whitelist correctly. Example of proper configuration： ![image](https://user-images.githubusercontent.com/37903575/229675919-cfb8ddce-fa5a-45a5-b625-6469a60b2e23.png) You should see the email address that you added to the whitelist.
> 
> ohh I saw my mistake, I didn't know that after writing the email I had to press the "Enter" key, if you don't press it and just write it and hit update it doesn't work, the correct steps would be
> 
> 1.- Write the email 2.- Press the enter key 3.- Click on update
> 
> THANKS!

**Reply by Root-FTW** [reply]
> I will optimize the operation later.
> 
> > > Have you checked the system settings -> whitelist again? It's possible that you didn't configure the whitelist correctly. Example of proper configuration： ![image](https://user-images.githubusercontent.com/37903575/229675919-cfb8ddce-fa5a-45a5-b625-6469a60b2e23.png) You should see the email address that you added to the whitelist.
> > 
> > 
> > ohh I saw my mistake, I didn't know that after writing the email I had to press the "Enter" key, if you don't press it and just write it and hit update it doesn't work, the correct steps would be
> > 1.- Write the email 2.- Press the enter key 3.- Click on update
> > THANKS!

Thank you !
